service/routes.py

```diff
@@ -296,36 +367,6 @@ def put(self):
 ###
```

Comment:
Could you uncomment the line 303, remove the comment on 302, and make the return statement
`return data, status.HTTP_201_CREATED, {'Location': location_url}` please? There is also a test case that needs to be uncommented that tests this header: `test_routes.py`, lines 90 and 91. I put those in, but since `WishlistResource` was not yet implemented it didn't work. Now that it's there - it should, and that's what the prof has in the example.